gulpfile.js

```diff
@@ -15,7 +16,7 @@ var config = {
 
 gulp.task('clean', function () {
   return gulp.src(config.paths.dist, {read: false})
```

Comment:
Actually initial code was not correct. Since we do not use piping here and need basic folder removal I believe we can use only `del` npm module. `vinyl-paths` sounds like an interesting touch but not for our use case. It is better we add only 1 dev-packages to rely on instead of 2.... Or if you can try to give it a shot https://github.com/gulpjs/gulp/blob/master/docs/recipes/delete-files-folder.md#delete-files-and-folders